### PR TITLE
Release 2021.11.0

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adding release notes for 2021.11.0
- **Is this PR related to a Monolith release?** Yes. Release notes 2021.11.0

### Resolves:
* [OKTA-441565](https://oktainc.atlassian.net/browse/OKTA-441565)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 1]
```diff
@@ -131,4 +131,26 @@ There is also additional information available in the `target`, such as `label`.
 
 See <GuideLink link="../customization-examples/#per-application-customization">Per-application customization</GuideLink> for an example of what you can do with request context.
 
+> **Note:** The following macros are only available when you enable the THEME_BUILDER feature.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:134]:
```suggestion
> **Note:** The following macros are only available when the Theme Builder feature is enabled in your org.
```

File: packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md [hunk 2]
```diff
@@ -4,6 +4,40 @@ title: Okta Identity Engine API Products Release Notes 2021
...
+| [Brands API support for auto-detecting contrast colors](#brands-api-support-for-auto-detecting-contrast-colors) | November 3, 2021 |
+| [New Devices API response property available](#new-devices-api-response-property-available) | November 3, 2021 |
+| [New error page macros for themed templates](#new-error-page-macros-for-themed-templates)                          | November 3, 2021          |
+| [Event Hooks daily limit](#event-hooks-daily-limit)                          | November 3, 2021          |
+| [Token-based SSO between native apps is now GA in Production](#token-based-sso-between-native-apps-is-now-ga-in-production)                          | October 6, 2021          |
+
+#### Brands API support for auto-detecting contrast colors
+
+The Brands API [Theme object properties](/docs/reference/api/brands/#theme-api-objects) `primaryColorContrastHex` and `secondaryColorContrastHex` automatically optimize the contrast between font color and the background or button color. The auto-detection feature can be disabled by updating either property value with an accepted contrast hex value. See [Update Theme](/docs/reference/api/brands/#update-theme).<!--OKTA-426715-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md:21]:
```suggestion
The Brands API [Theme object properties](/docs/reference/api/brands/#theme-api-objects) `primaryColorContrastHex` and `secondaryColorContrastHex` automatically optimize the contrast between font color and the background or button color. You can disable the auto-detection feature by updating either property value with an accepted contrast hex value. See [Update Theme](/docs/reference/api/brands/#update-theme).<!--OKTA-426715-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md [hunk 3]
```diff
@@ -4,6 +4,40 @@ title: Okta Identity Engine API Products Release Notes 2021
...
+
+Calls to the [List devices](/docs/reference/api/devices/#usage-example-expand-user) endpoint with an `expand=user` query now return the management status associated with each embedded user.<!--OKTA-431007-->
+
+#### New error page macros for themed templates
+
+Custom [error page templates](/docs/guides/custom-error-pages/use-macros/) include new macros to customize the URL (href) in addition to the button text for themed templates.<!--OKTA-440888-->
+
+#### Event Hooks daily limit
+
+The maximum allowable daily limit of Event Hooks for all orgs has increased from 100,000 to 200,000. A higher daily allocation of Event Hooks reduces the likelihood orgs will exceed their daily limits. See [Limits, duplicates, and order of Hook calls](/docs/reference/hooks-best-practices/#limits-duplicates-and-order-of-hook-calls).<!--OKTA-441433-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md:33]:
```suggestion
The maximum allowable daily limit of Event Hooks for all orgs has increased from 100,000 to 200,000. A higher daily allocation of Event Hooks reduces the likelihood that orgs will exceed their daily limits. See [Limits, duplicates, and order of Hook calls](/docs/reference/hooks-best-practices/#limits-duplicates-and-order-of-hook-calls).<!--OKTA-441433-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 4]
```diff
@@ -2,6 +2,85 @@
...
+| [Device Authorization grant type is now GA in Preview](#device-authorization-grant-type-is-now-ga-in-preview)                          | November 3, 2021          |
+| [Key pair additions to JWKS limited to 50 per client](#key-pair-additions-to-jwks-limited-to-50-per-client) | November 3, 2021 |
+| [Authentication API returns UD User Profile locale is GA in Production](#authentication-api-returns-ud-user-profile-locale-is-ga-in-production) | October 6, 2021 |
+| [Native SSO support is now GA in Production](#native-sso-support-is-now-ga-in-production)                          | October 6, 2021          |
+| [AES-GCM encryption support for SAML assertions is now GA in Production](#aes-gcm-encryption-support-for-saml-assertions-is-now-ga-in-production) | October 6, 2021 |
+| [Token-based SSO between native apps is now GA in Production](#token-based-sso-between-native-apps-is-now-ga-in-production) | October 6, 2021          |
+| [The Okta Org API is now GA in Production](#the-okta-org-api-is-now-ga-in-production)                          | March 3, 2021          |
+| [Bug fixed in 2021.11.0](#bug-fixed-in-2021-11-0)                          | November 3, 2021          |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:22]:
```suggestion
| [Bugs fixed in 2021.11.0](#bugs-fixed-in-2021-11-0)                          | November 3, 2021          |
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 5]
```diff
@@ -2,6 +2,85 @@
...
+
+#### Identity Providers API response includes ID token for generic OIDC provider
+
+The new response for the Identity Providers API [social authentication token operation](/docs/reference/api/idps/#social-authentication-token-operation) includes the addition of an ID token, if available, in addition to the access tokens.<!--OKTA-425470-->
+
+#### Brands API support for auto-detecting contrast colors
+
+The Brands API [Theme object properties](/docs/reference/api/brands/#theme-api-objects) `primaryColorContrastHex` and `secondaryColorContrastHex` automatically optimize the contrast between font color and the background or button color. The auto-detection feature can be disabled by updating either property value with an accepted contrast hex value. See [Update Theme](/docs/reference/api/brands/#update-theme).<!--OKTA-426715-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:30]:
```suggestion
The Brands API [Theme object properties](/docs/reference/api/brands/#theme-api-objects) `primaryColorContrastHex` and `secondaryColorContrastHex` automatically optimize the contrast between font color and the background or button color. You can disable the auto-detection feature by updating either property value with an accepted contrast hex value. See [Update Theme](/docs/reference/api/brands/#update-theme).<!--OKTA-426715-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 6]
```diff
@@ -2,6 +2,85 @@
...
+
+#### Device Authorization grant type is now GA in Preview
+
+Advancements in internet technology have seen an explosion of smart devices and the Internet of Things. Consumers need to sign in to applications that run on these devices, but the devices either lack support for a web browser or have limited ability for input, such as smart TVs, car consoles, and thermostats. As a result, users resort to insecure authentication solutions that are error-prone and time-consuming.
+
+The Device Authorization grant feature is an OAuth 2.0 grant type that allows users to sign in to input-constrained devices and also to devices that lack web browsers. This feature enables users to use a secondary device, such as a laptop or mobile phone, to complete sign-in to applications that run on such devices. See [Configure Device Authorization](/docs/guides/device-authorization-grant/main/).<!--OKTA-428052-->
+
+#### Key pair additions to JWKS limited to 50 per client
+
+The number of key pairs that can be added to a [JWKS (JSON Web Key Set)](/docs/guides/implement-oauth-for-okta-serviceapp/create-publicprivate-keypair/) can't exceed 50 per client.<!--OKTA-435434-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:48]:
```suggestion
The number of key pairs that can be added to a [JSON Web Key Set (JWKS)](/docs/guides/implement-oauth-for-okta-serviceapp/create-publicprivate-keypair/) can't exceed 50 per client.<!--OKTA-435434-->
```

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/use-macros/index.md [hunk 7]
```diff
@@ -79,4 +79,40 @@ Example:
...
+
+Example:
+
+```html
+ <a href="{{buttonHref}}" class="button">{{buttonText}}</a>
+```
+
+#### <span v-pre>`{{themedStylesUrl}}`</span>
+
+Inserts the href for the themed style sheet.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/use-macros/index.md:100]:
```suggestion
Inserts the URL for the themed style sheet
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 8]
```diff
@@ -2,6 +2,85 @@
...
+
+#### Native SSO support is GA in Production
+
+Single Sign-On (SSO) between browser-based web apps is achieved by leveraging shared cookies. Unlike web applications, native applications can't use web cookies. With Native SSO, Okta offers a token-based approach to achieve SSO between native applications.
+
+Native SSO allows you to protect native OpenID Connect applications, such as desktop apps and mobile apps, and achive SSO and Single Logout (SLO) between these applications. See [Configure SSO for native apps](/docs/guides/configure-native-sso/main/).<!--OKTA-435714-->
+
+#### AES-GCM encryption support for SAML assertions is now GA in Production
+
+To secure SAML assertions from attacks and to adopt a stronger security mechanism, Okta now supports AES128-GCM and AES256-GCM encryptions modes in addition to AES-128 and AES-256 for SAML applications. <!--OKTA-435717-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:62]:
```suggestion
To secure SAML assertions from attacks and to adopt a stronger security mechanism, Okta now supports AES128-GCM and AES256-GCM encryption modes in addition to AES-128 and AES-256 for SAML applications. <!--OKTA-435717-->
```

File: packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md [hunk 9]
```diff
@@ -8,54 +8,43 @@ As a developer building a custom application, you want to give your users the fr
...
+<a href='/docs/guides/identity-providers/' class='Button--blueDarkOutline card' data-proofer-ignore>
+	<span>Add an Identity Provider integration</span>
+</a>
 
 ## The big picture
 
 Okta manages connections to other Identity Providers for your application and sits between your application and the Identity Provider that authenticates your users.
 
-* When you use Okta as the [user store](https://developer.okta.com/product/user-management/) for your applications, users can sign in with their email and password by default. See our guides for how to sign in users to your [web](/docs/guides/sign-into-web-app/), [mobile](/docs/guides/sign-into-mobile-app/), and [single-page](/docs/guides/sign-into-spa/) apps.
+-   When you use Okta as the [user store](https://developer.okta.com/product/user-management/) for your applications, users can sign in with their email and password by default. See our guides for how to sign in users to your [web](/docs/guides/sign-into-web-app/), [mobile](/docs/guides/sign-into-mobile-app/), and [single-page](/docs/guides/sign-into-spa/) apps.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md:19]:
```suggestion
-   When you use Okta as the [user store](/docs/concepts/user-profiles/) for your applications, users can sign in with their email and password by default. See our guides for how to sign in users to your [web](/docs/guides/sign-into-web-app/), [mobile](/docs/guides/sign-into-mobile-app/), and [single-page](/docs/guides/sign-into-spa/) apps.
```

**Review by susanharper-okta — COMMENTED** [review]
a few things

**Review by vanngo-okta — COMMENTED** [review]
A few nits. Will continue with the review after a refresh.

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/use-macros/index.md [hunk 10]
```diff
@@ -79,4 +79,40 @@ Example:
...
 
+> **Note:** The following macros are only available when you enable the THEME_BUILDER feature.
+
+#### <span v-pre>`{{buttonText}}`</span>
+
+Inserts the button text based on the page context. The button takes the user back to the `buttonHref` when clicked. `{{back}}` is also supported for same purpose.
+
+#### <span v-pre>`{{buttonHref}}`</span>
+
+Inserts the href for the button.
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-error-pages/use-macros/index.md]: I'm unable to commit these suggestions here, for some reason. so i'll had them locally and push.

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚀

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚢

**Pull request merged**